### PR TITLE
Cleaned up CSS, Brought in NPM, Removed static css js files

### PR DESCRIPTION
**Pull request by Aghassi**
I'm submitting this because I would like a code review on what I have been doing. I spent the last few hours cleaning up the CSS to comply with CSSLint (which I will probably implement in the future using Gulp or Grunt). I have also:

* Removed the use of `!important` because it is not good practice. It forces cascading to break and defeats the purpose of the stylesheet. It can also lead to problems that are hard to debug. I did this in as many places as possible.
* Removed use of `#` in CSS and instead made classes for the IDs to find the items (good practice to have a class for everything).
* Updated `manifest.json` to support the `node_modules` directory
* Created `packages.json` for `npm` so we can have dependency management for our JS/CSS
* Updated `README` to add `npm install` step

The one visible change I have had with these changes is the color seems to have changed, but I am not worried about that because the functionality is still the same (though the game stream client sometimes freaks out, not sure if that was my doing). More to come as I work through things. Looking for any feedback on what I have right now. I won't push anymore until this is merged or hashed out.

### CODE REVIEW

**Comment by raidancampbell** [comment]
I'm not familiar with NPM.  It looks like you've made a file that lists all things moonlight-chrome depends on.  `npm install` would look at this file, then run off and install the dependencies?  What all is the [package.json](https://github.com/Aghassi/moonlight-chrome/blob/clean-up/package.json) file used for?

**Comment by Aghassi** [comment]
@raidancampbell Correct. NPM is Node Package Manager. `npm install` will look at the `package.json` file and install dependencies in a folder called `node_modules`. To install new dependencies, you just run `npm install *insert name here* --save` and it will add it to the package.json. You could tie this into the makefile probably so running `make` calls `npm install`, but you have to check and make sure the user has `npm` installed before running it. I'm also not too familiar with Makefiles, so I wouldn't know where to start with that.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moonlight-stream/moonlight-chrome.git"
+  },
+  "keywords": [
+    "moonlight",
+    "chrome",
+    "nvidia"
```
Comment (cgutman) [inline package.json]: Change to "gamestream"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "moonlight-chrome",
+  "version": "1.0.0",
+  "description": "Chrome extension for Moonlight, the open source Nvidia Stream app",
```
Comment (cgutman) [inline package.json]: Change to "Moonlight game streaming plugin for Google Chrome"

File: package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "type": "git",
+    "url": "git+https://github.com/moonlight-stream/moonlight-chrome.git"
+  },
+  "keywords": [
+    "moonlight",
+    "chrome",
+    "nvidia"
+  ],
+  "author": "Moonlight",
+  "license": "ISC",
```
Comment (cgutman) [inline package.json]: This is incorrect. Our license is GPLv3.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moonlight-stream/moonlight-chrome.git"
+  },
+  "keywords": [
+    "moonlight",
+    "chrome",
+    "nvidia"
+  ],
+  "author": "Moonlight",
```
Comment (cgutman) [inline package.json]: Author should be "Moonlight Game Streaming Project"

File: package.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/moonlight-stream/moonlight-chrome/issues"
+  },
+  "homepage": "https://github.com/moonlight-stream/moonlight-chrome#readme",
```
Comment (cgutman) [inline package.json]: What about linking to moonlight-stream.com?

**Comment by Aghassi** [comment]
@cgutman Changes should be reflected in the new commit I just pushed.

**Comment by raidancampbell** [comment]
Before merging this, I'd like to get the colors figured out, and fix the CSS spacing in the buttons: "Run Game", "Quit Current Game", and the game-select menu.  I'll have some time later this afternoon to help take a look

**Comment by cgutman** [comment]
Still not LGTM. Will leave specific feedback in a bit. Don't merge.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    "moonlight",
+    "chrome",
+    "gamestream"
+  ],
+  "author": "Moonlight Game Straming Project",
+  "license": "GPLv3",
+  "bugs": {
+    "url": "https://github.com/moonlight-stream/moonlight-chrome/issues"
+  },
+  "homepage": "moonlight-stream.com",
```
Comment (cgutman) [inline package.json]: Should we prefix with http:// ?

File: package.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moonlight-stream/moonlight-chrome.git"
+  },
+  "keywords": [
+    "moonlight",
+    "chrome",
+    "gamestream"
+  ],
+  "author": "Moonlight Game Straming Project",
```
Comment (cgutman) [inline package.json]: Typo

File: package.json (hunk 6 above)
Comment (Aghassi) [inline package.json]: Fixed. Will push next commit.

File: package.json (hunk 7 above)
Comment (Aghassi) [inline package.json]: Fixed. Will come in next commit.

**Comment by raidancampbell** [comment]
After much digging and more CSS than I'm comfortable with, MDL [notes](http://www.getmdl.io/started/)


     Using this method you will not be able to customize the color scheme of the MDL elements. 
     If you'd like to customize the color scheme prefer the CDN hosted or downloadable libraries. 
     In this case use our [Customize and Preview tool](http://www.getmdl.io/customize/index.html).

**Comment by cgutman** [comment]
Looks like this needs significant work if it's still applicable at all. Closing for now.

**Pull request closed without merging**